src/AutoRest.CSharp/Common/Output/Models/Requests/RestClientMethod.cs

```diff
@@ -34,6 +36,15 @@ public RestClientMethod(string name, string? summary, string? description, CShar
                 .Distinct()
                 .OrderBy(c => c.Code ?? c.Family * 100);
             ResponseClassifierType = new ResponseClassifierType(statusCodes);
+
+            PropertyBag = null;
+            // By default, we enable property bag feature in management plane and the real behavior will be determined later.
+            if (Configuration.AzureArm)
```

Comment:
@lirenhe Can we make a follow up issue to enable this in DPG?

Reply:
Created https://github.com/Azure/autorest.csharp/issues/3004 and @archerzz  will follow up